src/components/LicensePicker/utils.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+import parse from 'spdx-expression-parse'
+import isNil from 'lodash/isNil'
+import unset from 'lodash/unset'
+import get from 'lodash/get'
+import set from 'lodash/set'
+const NOASSERTION = 'NOASSERTION'
+
+// Shared methods appliable to License Picker
+export default class LicensePickerUtils {
+  static parseLicense(license) {
+    return license ? parse(license) : {}
+  }
+
+  // Returns a license string based on the rules in input, following the specification of https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60
+  static stringify(obj) {
```

Comment:
suggest `toString`.  also better name for `obj`. Here it seems to be `expression`?

Here and elsewhere